tap_xero/streams.py

```diff
@@ -87,7 +87,12 @@ def sync(self, ctx):
         offset = [self.tap_stream_id, "page"]
         start = ctx.update_start_date_bookmark(bookmark)
         curr_page_num = ctx.get_offset(offset) or 1
+
         filter_options = dict(since=start, order="UpdatedDateUTC ASC")
+        if self.tap_stream_id == "contacts":
```

Comment:
I'd like to see this implemented similarly to how it was attemptd in https://github.com/singer-io/tap-xero/pull/78. The stream-specific behavior should be encapsulated by the class.

Reply:
@KAllan357 Incorporated the requested changes